test/config_minneapolis_2013.json

```diff
@@ -1,7 +1,11 @@
 {
-  "auditOutput":"Minneapolis2013_audit_log.txt",
-  "visualizerOutput":"Minneapolis2013_visualizer.xlsx",
+  "auditOutput":"minneapolis_2013_audit_log.txt",
+  "visualizerOutput":"minneapolis_2013_visualizer.xlsx",
   "contestName":"2013 Minneapolis Mayor",
+  "jurisdiction":"Minneapolis",
+  "office":"Mayor",
+  "tabulateByPrecinct":true,
+  "precinctOutputDirectory":"test_output/minneapolis_2013_precincts",

```

Comment:
See above comment on `precinctOutputDirectory`. If we can fit it into this branch before merging, we should also add the `outputDirectory` field to each existing config file.